src/OrleansProviders/Storage/MemoryStorage.cs

```diff
@@ -56,7 +56,7 @@ public MemoryGrainStorage(string name, MemoryGrainStorageOptions options, ILogge
             this.grainFactory = grainFactory;
 
             //Init
-            logger.Info("Init: Name={0} NumStorageGrains={1}", name, this.options.NumStorageGrains);
+            logger.Info("Init: Name={Name} NumStorageGrains={NumStorageGrains}", name, this.options.NumStorageGrains);
```

Comment:
Use `LogInformation` instead of `Info` here and similarly elsewhere

Reply:
> Use `LogInformation` instead of `Info` here and similarly elsewhere

Done!